nixos/modules/services/x11/window-managers/ragnarwm.nix

```diff
@@ -0,0 +1,40 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.ragnarwm;
+in
+{
+  ###### interface
+
+  options = {
+    services.xserver.windowManager.ragnarwm = {
+      enable = mkEnableOption (lib.mdDoc "ragnarwm");
+      package = mkOption {
+        type        = types.package;
+        default     = pkgs.ragnarwm;
+        defaultText = literalExpression "pkgs.ragnarwm";
+        description = lib.mdDoc ''
+          ragnar package to use.
+        '';
```

Comment:
1. This should be a full sentence since this doesn't use `mkEnableOption` (which prepends some text to it).
2. I feel like either this or `enable` should mention that the main way of configuring this WM comes from overriding it with your own `config`.

Manual preview:
![image](https://github.com/NixOS/nixpkgs/assets/23431373/9ed87640-a29d-4f1e-8b90-48d1cc380818)